gender_novels/analysis/analysis.py

```diff
@@ -264,11 +277,12 @@ def dunning_total(m_corpus, f_corpus):
         wordcount_male = wordcounter_male[word]
         if word in wordcounter_female:
             wordcount_female = wordcounter_female[word]
-            dunning_result[word] = dunn_individual_word(totalmale_words,totalfemale_words,
-                                                        wordcount_male,wordcount_female)
-    dunning_result = sorted(dunning_result.items(), key = itemgetter(1))
-    print(dunning_result)
 
+            dunning_word = dunn_individual_word(totalmale_words, totalfemale_words,wordcount_male, wordcount_female)
+            dunning_result[word] = (dunning_word, wordcount_male, wordcount_female)
+    dunning_result = sorted(dunning_result.items(), key=itemgetter(1))
+
+    print(dunning_result)
```

Comment:
Update doctest to work with the current corpus numbers.  Be sure to run the doctest.  Thanks!